src/webauthn/server/mod.rs

```diff
@@ -295,6 +297,19 @@ impl CredentialCreationVerifier {
                 }
             }
 
+            Some(AttestationStatement::TPM(mut tpm)) => {
```

Comment:
```suggestion
            Some(AttestationStatement::TPM(tpm)) => {
```
Not mut anymore after changing `verify_public_key`